src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FulltextSearchResultsTab.java

```diff
@@ -44,39 +77,100 @@ public boolean shouldShow(BibEntry entry) {
 
     @Override
     protected void bindToEntry(BibEntry entry) {
-        if (!shouldShow(entry)) {
+        if (!shouldShow(entry) || entry == null) {
```

Comment:
Doesn't it make more sense for first check if the entry is null?

Reply:
Think so too. Java would then not even try to run shouldShow if entry is null. But the other question is, if javac is not automatically refactoring this...

Reply:
Haha yes of course. Such stupid mistakes. In this case it would not matter that much since shouldShow does not actually read the entry. It's just passed as shouldShow is inherited with that parameter.